src/components/landing-layout/navigation/sign-in-button.tsx

```diff
@@ -4,24 +4,40 @@
  * SPDX-License-Identifier: AGPL-3.0-only
  */
 
-import React from 'react'
+import equal from 'fast-deep-equal'
+import React, { useEffect, useState } from 'react'
 import { Button } from 'react-bootstrap'
 import { ButtonProps } from 'react-bootstrap/Button'
 import { Trans, useTranslation } from 'react-i18next'
 import { useSelector } from 'react-redux'
 import { LinkContainer } from 'react-router-bootstrap'
 import { ApplicationState } from '../../../redux'
 import { ShowIf } from '../../common/show-if/show-if'
+import { getApiUrl } from '../../../api/utils'
 
 export type SignInButtonProps = Omit<ButtonProps, 'href'>
 
+const INTERACTIVE_LOGIN_METHODS = ['internal', 'ldap', 'openid']
```

Comment:
Don't we have somewhere better to define this? I'm afraid it could get lost here…

Reply:
The only other location would the `api/auth`. I moved it there.